NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 ## Improvements
 
+* new tools to manage [collaborations](https://developer.box.com/reference/resources/collaboration/), `box_dir_invite`/`box_file_invite` is renamed `box_create_collab_dir`/`box_create_collab_file`, plus new functions to check existing collaborations (`box_get_collab_dir`/ `box_get_collab_file`) and delete collaborations (`box_delete_collab`).
```

Comment:
one more thing... I think our habit is to include the parentheses with the functions